pkg/clamav/scanner.go

```diff
@@ -42,7 +42,7 @@ func (scanner *clamScanner) Scan() ([]byte, error) {
 		return nil, err
 	}
 
-	err = ses.ScanPath(context.Background(), scanner.path)
+	err = ses.ScanPath(context.Background(), scanner.path, nil)
```

Comment:
would be nice if you can set custom context for this (in case this will get stuck or something you can set timeouts/etc.)